src/cloud/cdn/fastly-vcl-bypass-to-origin.md

```diff
@@ -1,14 +1,14 @@
 ---
 group: cloud-guide
-title: Custom VCL to bypass Fastly
+title: Custom VCL to bypass Fastly cache
 redirect_from:
    - /cloud/configure/fastly-vcl-badreferer.html
 functional_areas:
   - Cloud
   - Setup
 ---
 
-You can create a custom VCL snippet to bypass Fastly caching and submit requests directly to the origin server, for example to determine whether site issues are caused by caching, or to troubleshoot headers. You can configure the snippet to bypass Fastly for requests from a specific IP address or URL.
+You can create a custom VCL snippet to bypass Fastly caching and submit requests directly to the origin server, for example to determine whether site issues are caused by caching, or to troubleshoot headers. You can configure the snippet to bypass caching for requests from a specific IP address or URL.
```

Comment:
I would suggest "Fastly caching" as this eliminates the ambiguity between Fastly cache and Magento cache